docs/networking/best-practice.md

```diff
@@ -0,0 +1,351 @@
+    Jan 08 00:34:48 localhost kernel: tg3 0000:02:00.2 eno3: renamed from eth5 // eth5 / eno3 is not used by Harvester
+    Jan 08 00:34:48 localhost kernel: tg3 0000:02:00.1 eno2: renamed from eth3 // eth3 / eno2 is not used by Harvester
+    Jan 08 00:34:49 localhost kernel: i40e 0000:5d:00.0 eno5: renamed from eth0
+    Jan 08 00:34:49 localhost kernel: i40e 0000:af:00.0 ens4f0: renamed from eth4
+    Jan 08 00:34:49 localhost kernel: i40e 0000:5d:00.1 eno6: renamed from eth1
+    Jan 08 00:34:49 localhost kernel: i40e 0000:af:00.1 ens4f1: renamed from eth2
+    ```
+
+    The NIC driver of `eno5(0000:5d:00.0)` is `(intel) i40e 10Gbps Full Duplex`.
+    ```
+    $ grep "0000:5d:00.0" kernel.log
+
+    Jan 08 00:34:47 localhost kernel: i40e 0000:5d:00.0: fw 8.71.63306 api 1.11 nvm 10.54.7 [8086:1572] [103c:22fc]
+    Jan 08 00:34:47 localhost kernel: i40e 0000:5d:00.0: MAC address: 48:df:37:24:c2:00
+    Jan 08 00:34:47 localhost kernel: i40e 0000:5d:00.0: FW LLDP is enabled
+    Jan 08 00:34:47 localhost kernel: i40e 0000:5d:00.0 eth0: NIC Link is Up, 10 Gbps Full Duplex, Flow Control: None
+    Jan 08 00:34:47 localhost kernel: i40e 0000:5d:00.0: PCI-Express: Speed 8.0GT/s Width x8
+    Jan 08 00:34:47 localhost kernel: i40e 0000:5d:00.0: Features: PF-id[0] VFs: 64 VSIs: 66 QP: 112 RSS FD_ATR FD_SB NTUPLE DCB VxLAN Geneve PTP VEPA
+    Jan 08 00:34:49 localhost kernel: i40e 0000:5d:00.0 eno5: renamed from eth0
+    ```
+
+    The enabled NICs are detected.
+    ```
+    $ grep "is Up" kernel.log
+
+    Jan 08 00:34:47 localhost kernel: i40e 0000:5d:00.0 eth0: NIC Link is Up, 10 Gbps Full Duplex, Flow Control: None
+    Jan 08 00:34:48 localhost kernel: i40e 0000:5d:00.1 eth1: NIC Link is Up, 10 Gbps Full Duplex, Flow Control: None
+    Jan 08 00:34:48 localhost kernel: i40e 0000:af:00.0 eth4: NIC Link is Up, 10 Gbps Full Duplex, Flow Control: None
+    Jan 08 00:34:49 localhost kernel: i40e 0000:af:00.1 eth2: NIC Link is Up, 10 Gbps Full Duplex, Flow Control: None
+    ```
+
+### Enable Maintenance Mode
+
+1. (Optional) Stop VMs that cannot or must not be migrated.
+
+1. [Enable maintenance mode](../host/host.md#node-maintenance) on the target node to automatically migrate all VMs to other nodes.
+
+- Wait for everything to get ready, repeat the steps in [pre-check](#pre-check).
+
+- Stop a VM manually when:
+
+    - The VM fails to migrate
+
+    - The VM has some selectors which cause it has no other nodes to migrate
+
+    - The VM has special hardwares like pci-passthrough or vgpu and fails to migrate
+
+### (Optional) Update the VM Network
+
+:::note
+
+When the new Nics will be placed in different physical slots or those nics will have different uplink parameters, this step is mandatory.
+
+:::
+
+1. Check the node.
+
+    For a Harvester cluster node, when it is belonging to a VM Network (VlanConfig), the node object has a label with key `network.harvesterhci.io/vlanconfig`.
```

Comment:
I remember that VM Networks maps to NADs. It's Network Configs that map to VlanConfig. We might need to update the term here.